app/models/form_element.rb

```diff
@@ -30,7 +30,7 @@ def set_position
   def set_name
     unless name.blank? || ActionKitFields::ACTIONKIT_FIELDS_WHITELIST.include?(name)
       if !(name =~ ActionKitFields::VALID_PREFIX_RE) && !(name =~ /^(action_)+$/)
-        if data_type == 'paragraph'
+        if data_type == 'paragraph' or data_type == 'text'
```

Comment:
ruby `or` is for control flow, not for conditionals https://github.com/bbatsov/ruby-style-guide#no-and-or-or